apps/studio/src/components/tableview/RowFilterBuilder.vue

```diff
@@ -163,6 +163,15 @@
           </div>
           <div class="filter-add-apply">
             <div class="row fixed">
+               <button
+                v-if="filters.length > 1"
+                class="btn btn-flat btn-fab remove-filter"
+                type="button"
+                title="Remove filter"
+                @click="removeFilter(-1)"
+                >
+                  <i class="material-icons">remove</i>
+                </button>
```

Comment:
Hey @therealrinku, This is great! Could you also address the indentation warnings thrown by eslint?

You can do that by running `yarn workspace beekeeper-studio lint`. Thanks!

Reply:
Hi @azmy60!
sure, just pushed the changes 😄